benches/rust/benches/secure_cell_context_imprint_master_key.rs

```diff
@@ -90,7 +90,7 @@ fn pretty(size: usize) -> String {
     } else if size >= KB {
         format!("{} KB", size / KB)
     } else {
-        format!("{}", size)
+        format!("{size}")
```

Comment:
This would require a minimum supported version bump, see discussion in #984 As CI suggest, this does not build with Rust 1.56.

I'm not sure changing the supported platforms is something for the maintenance 0.14.x branch.

I think we should stick to eliminating warnings produced by 1.56 here.

Reply:
I agree. I will split it into a few separate.